pkg/istio-agent/sds-agent.go

```diff
@@ -453,4 +458,6 @@ func applyEnvVars() {
 	workloadSdsCacheOptions.InitialBackoffInMilliSec = int64(initialBackoffInMilliSecEnv)
 	// Disable the secret eviction for istio agent.
 	workloadSdsCacheOptions.EvictionDuration = 0
+	workloadSdsCacheOptions.AlwaysValidTokenFlag = alwaysValidTokenFlag
```

Comment:
Where is workloadSdsCacheOptions.AlwaysValidTokenFlag used?

Reply:
It is used in the key/cert rotation job: security/pkg/nodeagent/cache/secretcache.go:L557

Reply:
The following is the lines 556~562 in security/pkg/nodeagent/cache/secretcache.go, which does not include AlwaysValidTokenFlag.

			// Send the notification to close the stream if token is expired, so that client could re-connect with a new token.
			if sc.isTokenExpired() {
				cacheLog.Debugf("%s token expired", logPrefix)
				sc.callbackWithTimeout(connKey, nil /*nil indicates close the streaming connection to proxy*/)
				return true
			}

Reply:
It is in the method call `sc.isTokenExpired()`

Reply:
security/pkg/nodeagent/cache/secretcache.go:L830